gateway/1.8/modules/ROOT/pages/_partials/task-config-readiness-liveness.adoc

```diff
@@ -0,0 +1,191 @@
+//tag::config-readiness-liveness-page-body[]
+
+Flex Gateway supports two methods for ensuring reliability and efficiency.
+
+* Readiness probes test whether a gateway instance is configured correctly and ready for incoming traffic. This ensures that end-users are not affected by partially configured APIs. For information, refer to <<configure-a-readiness-probe>>.
+
+* Liveness probes test whether a gateway instance is operational. This minimizes downtime due to crashes or other failures. For information, refer to <<configure-a-liveness-probe>>.
+
+[[configure-a-readiness-probe]]
+== Configure a Readiness Probe
+
+To test whether a Flex Gateway instance is ready for incoming traffic, you can use the following CLI command:
+
+[source,ssh]
+----
+flexctl probe --check=readiness
+----
+
+If the Flex Gateway instance is ready to handle incoming traffic, the command returns an exit code of `0`. Otherwise the command returns an exit code of `1`.
```

Comment:
```suggestion
If the Flex Gateway instance is ready to process incoming traffic, the command returns an exit code of `0`. Otherwise, the command returns an exit code of `1`.
```
To simplify localization, I suggest this change but it could also be "manage" or "receive" in this context, please apply the term that is more accurate.